### PR TITLE
Added support for the RIKEN data center and associated Fugaku system

### PR DESCRIPTION
**Pull request by bvanessen**
using a contrib launcher.  This separates the Fugaku specific
parameters from the base PJM scheduler.

Tuning the proper MPI launch parameters for running on Fugaku.

### CODE REVIEW

**Comment by bvanessen** [comment]
No new failures
https://lc.llnl.gov/gitlab/vanessen/lbann/-/pipelines/158549

File: python/lbann/contrib/launcher.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import lbann
 import lbann.launcher
 from lbann.util import make_iterable, nvprof_command
+import re
```
Comment (benson31) [inline python/lbann/contrib/launcher.py:6]: I don't see where this is used.

File: python/lbann/launcher/pjm.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ def __init__(self,
                 (default: none).
             job_name (str, optional): Job name (default: none).
             partition (str, optional): Scheduler partition
-                (default: eap-small).
+                (default: small).
```
Comment (benson31) [inline python/lbann/launcher/pjm.py:44]: Isn't this RIKEN/Fugaku-specific?

**Review by benson31 — APPROVED** [review]
Seems good. There are a few debugging `print` statements still, but not the end of the world. I know you're still tweaking specifics, but the shell looks good.

**Review by benson31 — APPROVED** [review]
LGTM

**Pull request merged**